src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir-book";
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir-book", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.container {
+  display: block;
+  position: relative;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff;
+  width: 100%;
+}
+
+.container1 {
+  height: 70px;
+}
+
+.container2 {
+  height: 42px;
+}
+
+.image1 {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.image2 {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.input {
+  font-family: "Avenir-book", sans-serif;
```

Comment:
use `inherit` here for font-family
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit